docs/_layouts/default.html

```diff
@@ -35,7 +35,8 @@
     <div class="daptiv-container">
         {{ content }}
 
-        <footer class="footer">
+
+        <footer class="footer" style="margin: 25px 10px 2px 10px;">
```

Comment:
https://github.com/daptiv/style-guide/blob/master/docs/docs.scss Just need to add it here.